python/ql/test/query-tests/Functions/general/functions_test.py

```diff
@@ -198,3 +198,37 @@ def tuple_default(x=(1,2)):
 
 def safe_method(x=[]):
     return x.count(42)
+
+# Modification of parameter with default (sanitised)
```

Comment:
Very minor nit: we aim to use US English in our documentation, so get your `z`s ready for writing `sanitization` etc. :slightly_smiling_face: